daml-lf/spec/daml-lf-1.rst

```diff
@@ -4238,7 +4085,7 @@ Type Representation function
   Returns ``'True'`` if the first type representation is syntactically equal to
   the second one, ``'False'`` otherwise.
 
-  [*Available in versions = 1.7*]
+  [*Available in versions >= 1.7*]
```

Comment:
I think these type-specific comparison functions were remove in 1.dev? Same with the party literals comparison functions. Because of generic comparison.

```suggestion
  [*Available in version 1.7, 1.8*]
```

Reply:
good catch. I will fix that.